Task1/todo.js

```diff
@@ -0,0 +1,134 @@
+// This is the array that will hold the todo list items
+let todoItems = [];
+// This function will create a new todo object with text input, 
+//check to see which task is completed and id for remove
+
+// Render the todo items
+function renderTodo(todo)
+{
+    const list=document.querySelector("#todoList");
+    //select item which todo-id matches
+    const item=document.querySelector(`[todo-id='${todo.id}']`);
+    //delete the item
+    if(todo.deleted)
+    {
+        item.remove();
+        return
+    }
+
+    const node=document.createElement("li");
+    //create attribute as it uniquely define the key
+    node.setAttribute('todo-id',todo.id);
+    //creating delelte btn
+    const deleteButton =document.createElement("button");
+    deleteButton.classList.add('js-delete-todo');
+    deleteButton.innerHTML="Delete" 
+    //creating span 
+    node.innerHTML = `
+    <span>${todo.text}</span>
+    `;
+    //appending node and btn
+    node.appendChild(deleteButton)
+    list.appendChild(node)
+}
+
+//function to add item in an array
+function addTodo(text)
+{
+    const todo={
+        text,
+        id:Date.now(),
+    };
+
+    todoItems.push(todo);
+    //changed this 
+    todoItems.forEach((todo)=>{
+        renderTodo(todo);
+    });
+    // renderTodo(todo);
+}
+
+
+
+//function to delete each item
+function deleteTodo(idToBeRemoved)
+{
+
+    const index=todoItems.findIndex(({id})=>id===Number(idToBeRemoved));
```

Comment:
Try to keep the proper indentation in code everywhere for better readability. 

Also, the naming convention should be more descriptive of what a function or a variable is denoting/intent to perform. Example:

`const todoIndex = todoItems.findIndex(({ id }) => id === Number(idToBeRemoved));`

@Poorvi-3pillar